net/bitlbee/Makefile

```diff
@@ -0,0 +1,65 @@
+#
+# Copyright (C) 2006-2016 OpenWrt.org
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=bitlbee
+PKG_VERSION:=3.4.2
+PKG_RELEASE:=1
+
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
+PKG_SOURCE_URL:=http://get.bitlbee.org/src/
```

Comment:
Apparently bitlbee's site can be a bit dodgy, here's a few mirrors:

http://distcache.FreeBSD.org/local-distfiles/brix/
http://distcache.eu.FreeBSD.org/local-distfiles/brix/
http://distcache.us-east.FreeBSD.org/local-distfiles/brix/
http://distcache.us-west.FreeBSD.org/local-distfiles/brix/